verticaltabs.js

```diff
@@ -459,7 +457,17 @@ VerticalTabs.prototype = {
           tabsPopup.hidePopup();
         }
       }
-    });
+    };
+
+    leftbox.addEventListener('mouseenter', enter);
+    leftbox.addEventListener('mousemove', enter);
+    leftbox.addEventListener('mouseleave', exit);
+    let tab_context_menu = document.getElementById('tabContextMenu');
+    let toolbar_context_menu = document.getElementById('toolbar-context-menu');
```

Comment:
if I use `mouseenter` it doesn't count the children, so every time the mouse hovers over one of the list items it will close. so, unless I want to add mouse enter event to each child as well....